docs/bosh/README.md

```diff
@@ -199,19 +198,23 @@ Now you have the infrastructure ready to deploy a BOSH director.
 1. Create a **password-less** SSH key and upload the public component:
 
    ```
-   ssh-keygen -t rsa -f ~/.ssh/bosh -C bosh
+   ssh-keygen -t rsa -f ~/.ssh/bosh -C $service_account
```

Comment:
the SSH key and the service account aren't really related, either revert or a more descriptive comment?